devito/function.py

```diff
@@ -807,6 +808,9 @@ def __init__(self, *args, **kwargs):
             # Padding region
             self._padding = tuple((0, 0) for i in range(self.ndim))
 
+            self.interpolator = LinearInterpolator(self.point_symbols,
```

Comment:
I'm trying to progressively switch to private attributes (starting with _) whenever possible